Assets/Resources/PrototypePrefab/UIPLayer0.prefab

```diff
@@ -43,6 +43,7 @@ MonoBehaviour:
   leftInput: {x: 0, y: 0}
```

Comment:
Describe what this prefab is doing.
I don't remember any where this prefab is linked. 
Please check for consistency